db/main.go

```diff
@@ -0,0 +1,48 @@
+package db
+
+import (
+	"strings"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	// "github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+)
+
+
+// CreateDBIfNotCreated creates DynamoDB table if it doesn't exist
+func CreateDBIfNotCreated(tableName string) bool {
+	// Create DynamoDB client
+	sess := session.Must(session.NewSessionWithOptions(session.Options{
+		SharedConfigState: session.SharedConfigEnable,
+	}))
+	svc := dynamodb.New(sess)
```

Comment:
It would be worth exporting this as a public function. I would like to reuse this to quickly grab a configured client to compute my stats for the home